firestore.rules

```diff
@@ -48,6 +48,9 @@ service cloud.firestore {
       allow read: if request.auth.uid != null;
       allow write: if true;
     }
+    match /{path=**}/events/{event} {
+      allow read: if true;
+    }
```

Comment:
NOTE: This needs to be explicitly set for groupQuery to work as per the docs
https://firebase.google.com/docs/firestore/security/rules-query#secure_and_query_documents_based_on_collection_groups

Reply:
This should probably be `if request.auth.uid != null;` at a minimum. Currently any unauthenticated user on the internet could query this.